cytominer_eval/operations/enrichment.py

```diff
@@ -28,48 +30,54 @@ def enrichment(
     replicate_groups : List
         a list of metadata column names in the original profile dataframe to use as
         replicate columns.
-    percentile :  float
+    enrichment_percentile :  List of floats
```

Comment:
Done

Reply:
enrichment_percentile is still not the name of the function argument (line 19 above) do you see what I mean?

Reply:
I thinks it fine how it is now. Do you think I should change the name of the variable in enrichment.py to enrichment_percentile instead of just percentile?